### PR TITLE
style: snek case

### PR DESCRIPTION
**Pull request by banteg**
### What I did
- updated docs and examples with snek case
- changed js example of event polling to ape

### How I did it
wrote a script and edited by hand
```python
import os

import inflection
from rich import print
from textblob import TextBlob
from toolz import unique

targets = [('docs', '.rst'), ('examples', '.vy')]

for dir, ext in targets:
    for root, dirs, files in os.walk(dir):
        for file in files:
            if not file.endswith(ext):
                continue
            print(f"[green]{file}")
            source = open(f"{root}/{file}").read()
            text = TextBlob(source)
            for word in unique(text.words):
                is_camel = word == inflection.camelize(word, False).replace("-", "")
                is_snek = word == inflection.underscore(word)
                if is_camel and not is_snek:
                    print(f"  {word} -> {inflection.underscore(word)}")
                    source = source.replace(word, inflection.underscore(word))

            open(f"{root}/{file}", "wt").write(source)
```

### Commit message

Commit message for the final, squashed PR. (Optional, but reviewers will appreciate it! Please see [our commit message style guide](../../blob/master/docs/style-guide.rst#best-practices-1) for what we would ideally like to see in a commit message.)

### Description for the changelog
updated examples in documentation with snake_case

### CODE REVIEW

File: docs/compiler-exceptions.rst [hunk 1]
```diff
@@ -166,7 +166,7 @@ of the error within the code:
 
         vyper.exceptions.SyntaxException: line 4:20 invalid syntax
                 3 struct Bid:
-        ---> 4   blindedBid bytes32
+        ---> 4   blinded_bid bytes32
```
Comment (charles-cooper) [inline docs/compiler-exceptions.rst:169]:
```suggestion
        ---> 4   blinded_bid: bytes32
```

**Review by charles-cooper — APPROVED** [review]
i think this is ok (with a couple nits -- esp not having `@version` pragmas in examples)

i do wonder if having snake case in all the examples will confuse people who are trying to implement standard interfaces like ERC20

File: docs/compiler-exceptions.rst (hunk 1 above)
Comment (banteg) [reply docs/compiler-exceptions.rst:169]: it's an intentional error

Comment (charles-cooper) [reply docs/compiler-exceptions.rst:169]: Ah, got it

File: snek.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
+import os
```
Comment (fubuloubu) [inline snek.py:1]: You commit this by accident?

**Review by fubuloubu — APPROVED** [review]
Overall looks fine. I would put a note about style in the docs and mention that typically camelCase is used because that is how you would conform to ERC specs like ERC20

**Review by fubuloubu — APPROVED** [review]

**Comment by charles-cooper** [comment]
bumping this -- @banteg looks like there are some merge conflicts now as well as some test failures

**Comment by charles-cooper** [comment]
closing since this is not passing the test suite

**Pull request closed without merging**